src/pages/hero/Hero.jsx

```diff
@@ -0,0 +1,204 @@
+                      inStock,
+                      material,
+                      fastDelivery,
+                      offer,
+                    }) => {
+                      return (
+                        <div key={_id}>
+                          <Link to={`/product/${_id}`}>
+                            {" "}
+                            <div className="card_hr flex-center card_hero">
+                              <div className="card_hr_img card_img_hero">
+                                <img src={image} alt={productName} />
+                              </div>
+                              <div className="card_hr_content">
+                                <p title={name} className="h3 card_name_hero">
+                                  {name}
+                                </p>
+                                <StarRatings
+                                  rating={ratings}
+                                  starDimension="2rem"
+                                  starSpacing="0px"
+                                  starRatedColor="orange"
+                                />
+                                <div className="card_hr_price text_cl">
+                                  <p className="p-lg">
+                                    deal of the day:
+                                    <span className="h2 dod_price">
+                                      {` $${parseFloat(price)}`}
+                                    </span>
+                                  </p>
+                                  <p className="p-lg">
+                                    m.r.p.:
+                                    <span className="text_line_th">{`$${
+                                      Number(price) + Number(200)
+                                    }`}</span>
+                                  </p>
+                                </div>
+                              </div>
+                            </div>
+                          </Link>
+                        </div>
+                      );
+                    }
+                  )}
+              </div>
+            </div>
+          ))}
+        </div>
+        <div className=" banner_offer">
+          <img src="/assets/banner.png" alt="banner" />
+        </div>
+        <div className="text-center kodchasan hero_title text_cl">
+          <div className="h4">our best products </div>
+          <h2 className="h2">
+            <b>made only for you</b>
+          </h2>
+        </div>
+        {offerData.map((offer) => (
+          <div key={offer} id="head">
+            <div className="flex wrapper_head montserrat">
+              <h2 className="head_text">{offer}</h2>
+              <Link
+                to={`/product/${offer
+                  .replace(/ /g, "")
+                  .replace(/\d+% ?/g, "")
+                  .toLowerCase()}`}
+              >
+                <button className="btn-link view_all p">view all</button>
```

Comment:
Suggestion: We could extract out login from jsx and make it more cleaner and  easier to manage by using variable in jsx.

Reply:
Didn't get it @PuneetP16 can you elaborate?